docs/pages/inboxes/debug-your-app.md

```diff
@@ -63,10 +63,10 @@ These statistics are maintained per client instance, so each app installation ha
 
 ### Get aggregated statistics
 
-To return aggregated statistics, you can run:
+To return aggregated statistics, run:
 
-- Browser and Node SDKs: `client.apiAggregateStatistics()`
-- React Native, iOS, and Android SDKs: `client.debugInformation.aggregateStatistics` 
+- For Browser, Node, iOS, and Android SDKs: `client.debugInformation.apiAggregateStatistics()`
+- For React Native SDK: `client.debugInformation.aggregateStatistics`
```

Comment:
Bah - I know this was supposed to make these the same across SDKs - but I think React Native is using `aggregateStatistics` instead of `apiAggregateStatistics`? https://github.com/search?q=repo%3Axmtp%2Fxmtp-react-native+AggregateStatistics&type=code